modin/engines/ray/cudf_on_ray/frame/gpu_manager.py

```diff
@@ -62,21 +147,91 @@ def reduce(self, first, others, func, axis=0, **kwargs):
         return self.store_new_df(result)
 
     def store_new_df(self, df):
+        """
+        Store a new cudf_dataFrame in the dataframe_dict.
+        Iterate the current key int, and store the dataFrame in the dict
+        with this new iterated key
+        Return the key associated with this new cudf_dataFrame.
+        
+        Will be an OID corresponding to an int key.        
+
+        Parameters 
+        ----------
+            df : dataFrame
+                This is a dataFrame we're adding to cudf_dataframe_dict.
+        
+        Returns 
+        ------
+            self.key : int
+                This is the key associated the value dataFrame we passed in.
+
+        """
         self.key += 1
         self.cudf_dataframe_dict[self.key] = df
         return self.key
-
+    
     def free(self, key):
+        """
+        Free the dataFrame and associated key out of the dataframe_dict.
+
+        Parameters
+        ----------
+            key : int
+                The key we want to free (deletes the key-val pair).
+        """
         if key in self.cudf_dataframe_dict:
             del self.cudf_dataframe_dict[key]
 
+   
     def get_id(self):
-        return self.gpu_id
+        """
+        Get the gpu_id from the gpu_manager object.
 
-    def get_oid(self, key):
+        Returns
+        -------
+            self.gpu_id : int
+                the gpu_id from the gpu_manager object.
+                It will be represented as an OID, naturally.
+        """
+        return self.gpu_id
+   
+    def get_oid(self, key):         
+        """
+        Given a key, return the value of the key-val pair from cudf_dataframe_dict. 
+        
+        Parameters
+        ----------
+            key : int
+                The integer that corresponds to a particular key-value pair.
+        
+        Returns
+        -------
+            an oid corresponding to a cudf dataframe from cudf_dataframe_dict 
+            (wrapped up as an OID).
+        """
         return self.cudf_dataframe_dict[key]
 
+    
+
     def put(self, pandas_df):
-        if isinstance(pandas_df, pandas.Series):
+        """
+        Given a pandas_df object, 
+        convert it to a cudf_DataFrame, and add it to the cudf_dataframe_dict. 
+        Return the new key added to the dictionary (as an OID).
+        
+        Parameters
+        ----------
+            pandas_df : pandas_df
+                the pandas dataFrame object.
+                It may or may not be a pandas.Series object. 
+                If it is, convert it to a dataFrame.
+                Then, convert it to a cudf.dataFrame.
+
+        Returns
+        -------
+            an oid corresponding to the key generated 
+            when you added the new cudf_DataFrame object to the cudf_dataframe_dict.
+        """
+        if isinstance(pandas_df, pandas.Series): # if df instanceof a Pandas.series?
```

Comment:
# if df instanceof a Pandas.series? 

This could be included in the main dosctring. Just saying that pd.Series with be transform to DataFrames